libfprint/drivers/goodixtls/5395/goodix_device.c

```diff
@@ -258,3 +265,114 @@ gboolean fpi_goodix_device_reset(FpDevice *dev, guint8 reset_type, gboolean irq_
     GError *error = NULL;
     return fpi_goodix_device_send(dev, message, TRUE, 500, FALSE, &error);
 }
+
+gboolean fpi_goodix_device_gtls_connection(FpDevice *dev, GError *error)
```

Comment:
I think move gtls communication methods to another .c file (example: fpi_goodix_device_tls